pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -51,17 +53,17 @@ let
 in
 buildPythonPackage {
   pname = "tensorflow" + lib.optionalString cudaSupport "-gpu";
-  inherit (packages) version;
+  version = packages."${"version" + lib.optionalString (cudaSupport && cudaPackages.cudaFlags.isJetsonBuild) "_jetson"}";
```

Comment:
Using `isJetsonBuild` in quite a few places, maybe it's ok to inherit it in a let in before the derivation

Reply:
Done